instrumentation/jvm-metrics/src/main/java/com/splunk/opentelemetry/instrumentation/jvmmetrics/micrometer/MicrometerAllocatedMemoryMetrics.java

```diff
@@ -19,6 +19,7 @@
 import static com.splunk.opentelemetry.instrumentation.jvmmetrics.AllocatedMemoryMetrics.METRIC_NAME;
 
 import com.splunk.opentelemetry.instrumentation.jvmmetrics.AllocatedMemoryMetrics;
+import io.micrometer.common.lang.NonNull;
```

Comment:
wrong import?

Reply:
Nope, micrometer has their own. 🙄

Reply:
why on earth would you want to use the micrometer one?

Reply:
This class is an implementation of a micrometer interface. That interface uses the micrometer specific annotation. Using an annotation that doesn't match the actual interface declaration seems hacky.